app/views/contact_points/index.html.haml

```diff
@@ -0,0 +1,11 @@
+%h1 Contact points
+
+%ul
+  - @contact_points.each do |contact_point|
+    %li
+      %p= "Email: #{contact_point.email}" unless contact_point.email == ''
```

Comment:
@mattrayner yes, it is now in a partial using `present?` helper, thanks for the tip 🙂